<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryEdit</class>
 <widget class="QWidget" name="EntryEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>703</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>703</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="title_edit">
      <property name="maxLength">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QSpinBox" name="tracks_number_spin_box">
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="12" column="2">
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="date_spin_box"/>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Seconds</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="minutes_spin_box">
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QSpinBox" name="discs_number_spin_box">
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Tracks number</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Lable</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Minutes</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="singer_edit">
      <property name="maxLength">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Duration</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Release date</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Discs number</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="seconds_spin_box">
      <property name="maximum">
       <number>59</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Singer</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QTextEdit" name="desctiption_edit"/>
    </item>
    <item row="9" column="1" colspan="2">
     <widget class="QLineEdit" name="lable_edit">
      <property name="maxLength">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Genre</string>
      </property>
     </widget>
    </item>
    <item row="12" column="3">
     <widget class="QPushButton" name="ok_button">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Studio</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>300</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="1" colspan="2">
     <widget class="QLineEdit" name="studio_edit">
      <property name="maxLength">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="2">
     <widget class="QLineEdit" name="genre_edit">
      <property name="maxLength">
       <number>1000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
